collect_app/src/main/java/org/odk/collect/android/preferences/ServerPreferencesFragment.java

```diff
@@ -67,12 +65,14 @@
 import static org.odk.collect.android.preferences.GeneralKeys.KEY_SELECTED_GOOGLE_ACCOUNT;
 import static org.odk.collect.android.preferences.GeneralKeys.KEY_SMS_GATEWAY;
 import static org.odk.collect.android.preferences.GeneralKeys.KEY_SMS_PREFERENCE;
-//import static org.odk.collect.android.preferences.GeneralKeys.KEY_SUBMISSION_TRANSPORT_TYPE;
 import static org.odk.collect.android.preferences.GeneralKeys.KEY_SUBMISSION_URL;
-import static org.odk.collect.android.utilities.DialogUtils.showDialog;
 import static org.odk.collect.android.preferences.GeneralKeys.KEY_TRANSPORT_PREFERENCE;
+import static org.odk.collect.android.utilities.DialogUtils.showDialog;
 import static org.odk.collect.android.utilities.gdrive.GoogleAccountsManager.REQUEST_ACCOUNT_PICKER;
 
+//import android.preference.ListPreference;
+//import static org.odk.collect.android.preferences.GeneralKeys.KEY_SUBMISSION_TRANSPORT_TYPE;
```

Comment:
(I think you removed different ones than these)

Reply:
You are totally correct. My bad. Will fix now - I'll leave the commented out fields deleted as well.

Reply:
Actually it looks like the commented out lines have something to do with the recent disabling of SMS features (https://github.com/opendatakit/collect/commit/d47e3312ae956f7af421505b44e665d5a181e925#diff-28149f9be7f929447487b6010fea53c3). 

Definitely happy to just delete the commented code (its in version control we can get it back) but maybe we should leave it alone in this PR? I can revert the delete of the comment fields if so.

Reply:
I see. Yeah I guess that should be reverted then. Thanks for clarifying. Hopefully all that code can just be removed soon.

Reply:
@cooperka reverted!